packages/web-app/src/containers/configureLoanOffer/index.tsx

```diff
@@ -223,14 +224,29 @@ const ConfigureLoanOfferForm: React.FC<ConfigureLoanOfferFormProps> = ({
                         fieldState: { error },
                     }) => (
                         <>
-                            <StyledInput
-                                mode={error ? 'critical' : 'default'}
-                                name={name}
-                                type="text"
-                                value={value}
-                                placeholder="0x..."
-                                onBlur={onBlur}
-                            />
+                            {collateralType as CollateralType === 'ERC20' ? (
+                                <StyledSelect
+                                    name={name}
+                                    value={value}
+                                    onChange={onChange}
+                                    defaultValue={""}
+                                >
+                                    <option value="" disabled>{t('labels.selectAnOption')}</option>
+                                    {collateralTypeOptions.map((item) => (
```

Comment:
It should be SUPPORTED_TOKENS[SupportedNetwork.MUMBAI].map ??

Reply:
Yes. My bad.